packages/flab/LAB.src.js

```diff
@@ -8,6 +8,7 @@
 
 		// constants for the valid keys of the options object
```

Comment:
It's tough for me to tell how the options object is set for LABjs. Looks like we pass just a couple of parameters into the LABjs `globaldefaults` and let our typical defaults dictate all of the other options. This is a nit pick, but I find that flow a bit confusing. It's certainly the most line-efficient way to go about it, though, and if it doesn't matter most of the time then feel free to ignore this.